test/one/profiler/test/Assert.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright The async-profiler authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package one.profiler.test;
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+public class Assert {
+
+    public static void contains(Output out, String regex) throws AssertionError {
+        if (!out.contains(regex)) {
+            throw new AssertionError("Expected: " + regex + "\nReceived out may be available in build/test/logs.");
```

Comment:
Multi-line exception messages are discouraged. Repeating "Received out may be available in build/test/logs." in each error message is redundant. TestRunner could print this instead.

Reply:
Addressed. Will reflect in next commit